crates/generator/src/syscalls/mod.rs

```diff
@@ -602,4 +619,19 @@ impl<'a, S: State, C: ChainView> L2Syscalls<'a, S, C> {
         self.redir_log_handler.append_log(s);
         Ok(())
     }
+
+    fn get_syscall_cycles(syscall: u64, cycles_config: &SyscallCyclesConfig) -> u64 {
+        match syscall {
+            SYS_STORE => cycles_config.sys_store_cycles,
+            SYS_LOAD => cycles_config.sys_load_cycles,
+            SYS_CREATE => cycles_config.sys_create_cycles,
+            SYS_LOAD_ACCOUNT_SCRIPT => cycles_config.sys_load_account_script_cycles,
+            SYS_STORE_DATA => cycles_config.sys_store_data_cycles,
+            SYS_LOAD_DATA => cycles_config.sys_load_data_cycles,
+            SYS_GET_BLOCK_HASH => cycles_config.sys_get_block_hash_cycles,
+            SYS_RECOVER_ACCOUNT => cycles_config.sys_recover_account_cycles,
+            SYS_LOG => cycles_config.sys_log_cycles,
+            _ => 0,
```

Comment:
I think we should panic here, return `0` may ignoring unintended behaviours

Reply:
But we only include partial syscalls in `SyscallCyclesConfig`.
0 is intended default value for syscalls not in config.